### PR TITLE
fix CJK characters width in output

### PR DESCRIPTION
**Pull request by sophgn**

### CODE REVIEW

**Comment by sophgn** [comment]
5a03bc20f247c58c
before and after

File: pre_commit/commands/run.py [hunk 1]
```diff
@@ -33,8 +34,12 @@
 logger = logging.getLogger('pre_commit')
 
 
+def _len_cjk(msg: str) -> int:
+    return len(msg) + sum(east_asian_width(c) in 'FW' for c in msg)
```
Comment (asottile) [inline pre_commit/commands/run.py]: I would prefer `in {'F', 'W'}`

File: pre_commit/commands/run.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 from typing import Sequence
 from typing import Set
 from typing import Tuple
+from unicodedata import east_asian_width
```
Comment (asottile) [inline pre_commit/commands/run.py]: I tend to prefer `import unicodedata` and then `unicodedata.east_asian_width` for stdlib modules

File: pre_commit/commands/run.py [hunk 3]
```diff
@@ -13,9 +13,9 @@
 from typing import Sequence
 from typing import Set
 from typing import Tuple
-from unicodedata import east_asian_width
 
 from identify.identify import tags_from_path
+from wcwidth import wcswidth
```
Comment (asottile) [inline pre_commit/commands/run.py]: `wcwidth` introduces a dependency, I'd rather not -- `unicodedata` is fine for now (this is only accidentally working due to `pytest`)

**Pull request merged**